paasta_tools/paasta_cli/cmds/status.py

```diff
@@ -200,9 +213,20 @@ def paasta_status(args):
         cluster_filter = args.clusters.split(",")
     else:
         cluster_filter = None
+    if args.instances is not None:
+        instance_filter = args.instances.split(",")
+    else:
+        instance_filter = None
```

Comment:
can the default here be [] ? That way you don't have to test for None everywhere